website/docs/docs/cloud-integrations/semantic-layer/power-bi.md

```diff
@@ -0,0 +1,154 @@
+
+These tables do not actually map to an underlying table in your data warehouse. Instead, Power BI sends queries to these tables and (before actually executing on the warehouse) the Semantic Layer servers:
+- Parse the SQL.
+- Extract all the queried columns, group bys and filters.
+- Generates SQL to query your existing tables. 
+- Returns data back to Power BI, which doesn’t know any of this happened.
+
+<Lightbox src="/img/docs/cloud-integrations/sl-pbi/sl-pbi.jpg" width="90%" title="Power BI integration diagram" />
+
+This allows for very flexible analytics workflows, like drag and drop metrics and slice by dimensions and entities &mdash; the Semantic Layer will generate the appropriate SQL to actually query your data source for you.
+
+## Considerations
+
+<Expandable alt_header="Not every “column” of METRICS.ALL are compatible with every other column">
+
+- `METRICS.ALL` combines all your existing metrics, entities and dimensions. Queries must be valid Semantic Layer queries, otherwise they'll fail with MetricFlow query compilation errors.
+
+- For saved query tables, all “columns” will be compatible with every other “column” since, by definition, saved queries are valid queries that can be sliced by any of the dimensions present in the query.
+</Expandable>
+
+<Expandable alt_header="The dbt Semantic Layer connector does not support Import mode natively">
+
+- Use `DirectQuery` mode to ensure compatibility.
+- `Import` mode tries to select an entire table to import into Power BI, which means it'll likely generate SQL that translates to an invalid Semantic Layer query which will try to query all metrics, dimensions and entities at the same time.
+- To import data into a PowerBI report:
+   - Select a valid combination of columns to import, (something that will generate a valid Semantic Layer query)
+   - You can use `Table.SelectColumns` for this: `= Table.SelectColumns(Source{[Item="ALL",Schema="METRICS",Catalog=null]}[Data], {"Total Profit", "Metric Time (Day)"})`
+   - Be aware that all calculations will happen inside of Power BI and won’t pass through Semantic Layer servers. This could lead to incorrect or diverging results.
+   - For example, the Semantic Layer is usually responsible for rolling up cumulative metrics to coarser time granularities. Doing a sum over all the weeks in a year to get a yearly granularity out of a weekly Semantic Layer query will most likely generate incorrect results. Instead, you should query the Semantic Layer directly to get accurate results.
+
+</Expandable>
+
+<Expandable alt_header="The dbt Semantic Layer connector ignores aggregations defined in Power BI">
+
+- If you change the aggregation type of a metric from `SUM()` to `COUNT()` or anything else, nothing will change. This is because aggregation functions are defined in the Semantic Layer and we ignore them when translating Power BI generated SQL into Semantic Layer queries.
```

Comment:
We'll want to add another note mentioning that:
- `Count (Distinct)`, `Standard Deviation`, `Variance`, and `Median` might all throw an error and not work at all